setuptools/dist.py

```diff
@@ -418,7 +418,10 @@ def _finalize_license_files(self) -> None:
             patterns = ['LICEN[CS]E*', 'COPYING*', 'NOTICE*', 'AUTHORS*']
 
         self.metadata.license_files = list(
-            unique_everseen(self._expand_patterns(patterns))
+            map(
+                lambda path: path.replace("\\", "/"),
```

Comment:
Should we use `os.sep` here instead of `\\` (e.g. if somehow `\\` ends showing up on a linux build, we want the build to crash right, because it would mean incorrect input from the user)?

Reply:
I don't believe that's necessary here. `path` isn't a user input but rather the glob matched license path from `self._expand_patterns`. The idea with this line is just to normalize the separator since the core metadata field only allows forward `/`.

Reply:
Not sure I've ever seen that in the wild but you're right it's a possibility. Pushed https://github.com/pypa/setuptools/pull/4728/commits/3e9b9c7fbe4b753bbc80ee8be01c9a8a9233970d to use `os.sep`.